README.md

```diff
@@ -78,7 +78,8 @@ var errorHandler = require('@google/cloud-errors')({
 ```JS
 var express = require('express');
 var app = express();
-var errorHandler = require('@google/cloud-errors')();
+// Will create a errorHandler instance based off env variables
+var errorHandler = require('@google/cloud-errors');
```

Comment:
I don't think we should be highlighting any API other than `.start`. I think it would be *nice* that we would help/warn the user if they forgot to call `start`. I would like to keep the API surface area as minimal as possible.